frontend/src/reviews/ReviewCommentDisplay.tsx

```diff
@@ -1,21 +1,38 @@
 import { Menu as MenuIcon } from '@mui/icons-material'
-import { Box, Card, Divider, IconButton, Menu, MenuItem, Stack, Typography } from '@mui/material'
+import { Box, Button, Card, Divider, IconButton, Menu, MenuItem, Stack, Typography } from '@mui/material'
+import { useTheme } from '@mui/material/styles'
+import { patchResponse } from 'actions/response'
 import { useState } from 'react'
 import MarkdownDisplay from 'src/common/MarkdownDisplay'
+import RichTextEditor from 'src/common/RichTextEditor'
 import UserAvatar from 'src/common/UserAvatar'
 import UserDisplay from 'src/common/UserDisplay'
-import { EntityKind, ResponseInterface } from 'types/types'
-import { formatDateString } from 'utils/dateUtils'
+import { EntityKind, ResponseInterface, User } from 'types/types'
+import { formatDateString, formatDateTimeString } from 'utils/dateUtils'
+import { getErrorMessage } from 'utils/fetcher'
 
 type ReviewCommentDisplayProps = {
   response: ResponseInterface
   onReplyButtonClick: (value: string) => void
+  currentUser: User | undefined
+  mutateResponses: () => void
 }
 
-export default function ReviewCommentDisplay({ response, onReplyButtonClick }: ReviewCommentDisplayProps) {
+export default function ReviewCommentDisplay({
+  response,
+  onReplyButtonClick,
+  currentUser,
+  mutateResponses,
+}: ReviewCommentDisplayProps) {
   const [entityKind, username] = response.entity.split(':')
 
+  const theme = useTheme()
+
   const [anchorEl, setAnchorEl] = useState<null | HTMLElement>(null)
+  const [isEditMode, setIsEditMode] = useState(false)
+  const [updatedComment, setUpdatedComment] = useState('')
```

Comment:
With these kinds of editable inputs I find it's easier to always use one value, rather than alternating between using `release.comment` and `updatedComment` depending on the situation.

Consider renaming this to `comment` and using it throughout. You'll then just need to set it to either `release.comment` (on initial render and when cancelling edit mode) or let the `RichTextEditor` control it when in edit mode.